include/zephyr/bluetooth/conn.h

```diff
@@ -730,7 +744,7 @@ int bt_conn_set_security(struct bt_conn *conn, bt_security_t sec);
  *
  *  @return Connection security level
  */
-bt_security_t bt_conn_get_security(struct bt_conn *conn);
+bt_security_t bt_conn_get_security(const struct bt_conn *conn);
```

Comment:
I wonder if we should update the release notes for these as well since we are technically modifying a stable API